src/pages/album/components/Content.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```

Comment:
Please remove the whole `Content.js` component, you don't need it. Use `<SongList/>` directly.